test/bootstrap.php

```diff
@@ -17,3 +18,6 @@
 require_once 'src/wovnio/modified_vendor/SimpleHtmlDomNode.php';
 
 require_once 'test/helpers/StoreAndHeadersFactory.php';
+
+// disable error logging
+\Wovnio\Wovnphp\Logger::set(new \Wovnio\Wovnphp\Logger(true));
```

Comment:
could we just have a `quite` method to disable logger output?, then we don't really need `set` method

Reply:
I still want `set` method for the reason I mentionned in the above comment. 

I agree with `quiet` function, I’ll add it. What name should I use for “unquiet” function?

Reply:
`unquite` is a good name imo 👯‍♂️

Reply:
I made the change. In the end I went for simple setter.